packages/nextjs/utils/scaffold-stark/contractsData.ts

```diff
@@ -5,3 +5,27 @@ export function getAllContracts() {
   const contractsData = contracts?.[scaffoldConfig.targetNetworks[0].network];
   return contractsData ? contractsData : {};
 }
+
+function bigIntToHex(bigInt: bigint) {
+  return bigInt.toString(16);
+}
+
+function hexToUtf8(hexStr: string) {
+  let bytes = [];
+  for (let i = 0; i < hexStr.length; i += 2) {
+    bytes.push(parseInt(hexStr.substring(i, i + 2), 16));
+  }
+  try {
+    return new TextDecoder("utf-8").decode(new Uint8Array(bytes));
+  } catch (e) {
+    console.error("Failed to decode:", e);
+    return "";
+  }
+}
+
+export function decodeBigIntArrayToText(byteArray: string[]) {
```

Comment:
i think we can just remove the `import decode ...` from the `MyHoldings.tsx` file.

Reply:
we dont need to define this function.

Reply:
@huisq can you update this PR please ? thanks 🙌